core/dictgen/src/rootcling_impl.cxx

```diff
@@ -2841,8 +2841,8 @@ int CheckClassesForInterpreterOnlyDicts(cling::Interpreter &interp,
 ////////////////////////////////////////////////////////////////////////////////
 /// Make up for skipping RegisterModule, now that dictionary parsing
 /// is done and these headers cannot be selected anymore.
-
-int FinalizeStreamerInfoWriting(cling::Interpreter &interp, bool writeEmptyRootPCM=false)
+///
+int MaybeEmitROOTPCM(cling::Interpreter &interp, bool writeEmptyRootPCM=false)
```

Comment:
Why "maybe"?

Reply:
Hm... that's probably a leftover of something else. Thanks!